app/src/config/service.js

```diff
@@ -0,0 +1,7 @@
+exports.load = () => ({
+  name: process.env.SERVICE_NAME || 'NotSet',
+  env: process.env.SERVICE_ENV || 'NotSet',
+  functionName: process.env.AWS_LAMBDA_FUNCTION_NAME || 'NotSet',
```

Comment:
function name can be taken from context (passed to the function as a parameter), same as version

Reply:
done